3501/0/about_strings.rb

```diff
@@ -0,0 +1,204 @@
+# rubocop:disable Lint/MissingCopEnableDirective, Metrics/ClassLength
```

Comment:
u must enable cops which were disabled previously instead of disabling this warning